src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
-export interface Figure {
+export type FigureColor = 'red' | 'green' | 'blue';
+export type FigureShape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
To avoid duplicating of figure names, will be better to create Enum from this type